src/js/stores/CosmosPackagesStore.js

```diff
@@ -263,6 +281,15 @@ class CosmosPackagesStore extends GetSetBaseStore {
     return null;
   }
 
+  getPackageVersions(packageName) {
+    const packagesVersions = this.get("packagesVersions");
```

Comment:
I know that this method is following the pattern `getPackageDetails` is using to fetch and store data and thus I'm ok moving forward with the current implementation, but we must refactor this store! Storing and accessing the data in this way is going to create issues sooner than later as there is a very high chance that the data is mutated in unintended ways and there is no guarantee the versions match the package.

Reply:
@weblancaster and I talked about this because we recognize the store is a mess. This is one of the first stores I think dcos-ui had. Perfect example of where we should file an issue that we can follow up on, but the scope of adjustment shouldn't be included in this feature work 👍

Reply:
@weblancaster @MatApple can one of you please file an issue?